addons/account_edi_ubl_cii/models/account_edi_xml_cii_facturx.py

```diff
@@ -293,7 +293,7 @@ def _import_fill_invoice_form(self, invoice, tree, qty_factor):
 
         # ==== Invoice origin ====
 
-        invoice_origin_node = tree.find('./{*}OrderReference/{*}ID')
+        invoice_origin_node = tree.find('./{*}OrderReference/{*}ID') or tree.find('.//{*}BuyerOrderReferencedDocument/{*}IssuerAssignedID')
```

Comment:
IDK what is the stable policy for the accounting. We could remove the `./{*}OrderReference/{*}ID` but maybe it would break a client custo.. I propose to keep it and remove it in the master forward port.

Reply:
Since it's incorrect, I don't think we should worry about keeping the previous path.